OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDetailFragment.java

```diff
@@ -304,8 +305,8 @@ private void configureEnrollButton() {
         if (mEnrolled) {
             mEnrollButton.setText(R.string.view_course_button_text);
         } else if (courseDetail.invitation_only) {
+            mEnrollButton.setBackground(ContextCompat.getDrawable(getContext(), R.drawable.edx_invitation_only_button));
```

Comment:
Instead of creating a new drawable we can update the `drawable/edx_enroll_button.xml` to match the implementation done in `drawable/edx_footer_button.xml`.

For usage example you can look @ `layout/create_new_item_layout.xml`'s implementation and usage.